### DISCUSSION TITLE
T2i Semantic Segmentation Color Reference Chart - v21

### DISCUSSION THREAD

**Discussion by AugmentedRealityCat** [TencentARC/T2I-Adapter#25] [Show and tell]
This document presents the colors associated with the 182 classes of objects recognized by the T2i Semantic Segmentation model.

This updated version (v22) fixes an error with the HEX color codes, which had an extra 0 that has now been removed.

pdf version
[T2i_Segmentation_Color_Ref_v22.pdf](https://github.com/TencentARC/T2I-Adapter/files/10954172/T2i_Segmentation_Color_Ref_v22.pdf)

spreadsheet in LibreOffice ods format
[T2i_Segmentation_Color_Ref_v22.ods](https://github.com/TencentARC/T2I-Adapter/files/10954174/T2i_Segmentation_Color_Ref_v22.ods)

![T2i_Segmentation_Color_Ref_v22](https://user-images.githubusercontent.com/114955776/224622415-9f032b62-3ef2-4406-9f00-060a8ee0aeb4.png)

**Comment by hadipash** [comment]
Thanks for your effort. Additionally, [here](https://github.com/PITI-Synthesis/PITI/blob/main/preprocess/preprocess_mask.py)'s code to convert IDs into RGB values (as mentioned by the [authors](https://github.com/TencentARC/T2I-Adapter/issues/35#issuecomment-1463265191))
